### PR TITLE
added asset details to query liquidity provider response

### PR DESCRIPTION
**Pull request by Jedi2002**
image

### CODE REVIEW

File: x/clp/handler.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package clp
 
 import (
 	"fmt"
+	clpkeeper "github.com/Sifchain/sifnode/x/clp/keeper"
```
Comment (AustinoBombino) [inline x/clp/handler.go:5]: I'm a bit hazy on the inclusion of clpkeeper here. Do we not already have the keeper passed as a parameter to the handler?

Comment (Jedi2002) [reply x/clp/handler.go:5]: I moved calculations.go to the keeper package .
We have alwasys used the keeper package in handler.go ,just that I had created alias for the functionss ( see alias.go) ,  alias.go and handler.go are both in the same package .

Comment (Jedi2002) [reply x/clp/handler.go:5]: I can alias these funtions as well 
keeper.CalculatePoolUnits
keeper.SwapOne

but I am not sure if it makes sense for us to expose them to be used through a direct clp import .

Comment (AustinoBombino) [reply x/clp/handler.go:5]: If the calculations.go functions are now in the keeper package then can't they be called from the keeper passed to the handler?
 ie: 
 `func handleMsgCreatePool(ctx sdk.Context, keeper Keeper, msg MsgCreatePool)`
 
 I'm not trying to suggest adding them to the alias, I'm just confused about the two instances of the keeper. Is the clpKeeper different from the keeper passed as a parameter above?

File: x/clp/keeper/calculations.go [hunk 2]
```diff
@@ -180,3 +180,11 @@ func calcSwapResult(X, x, Y sdk.Uint) sdk.Uint {
 	denom := d.Mul(d)
 	return (x.Mul(X).Mul(Y)).Quo(denom)
 }
+
+func CalcualteAllAssetsforLP(pool types.Pool, lp types.LiquidityProvider) (sdk.Uint, sdk.Uint, sdk.Uint, sdk.Uint) {
```
Comment (AustinoBombino) [inline x/clp/keeper/calculations.go:184]: Just a typo here: Calcualte -> Calculate

File: x/clp/handler.go (hunk 1 above)
Comment (Jedi2002) [reply x/clp/handler.go:5]: The calculations.go were moved to the keeper package , but they not part part of the keeper struct .

Comment (AustinoBombino) [reply x/clp/handler.go:5]: Okay gotcha, that answers my question. Sorry for the confusion.

**Review by utx0 — APPROVED** [review]

**Pull request merged**